src/CommunityToolkit/Xamarin.CommunityToolkit/Extensions/VisualElement/VisualElementExtension.shared.cs

```diff
@@ -64,5 +65,74 @@ public static void AbortAnimations(this VisualElement element, params string[] o
 
 		internal static bool TryFindParentOfType<T>(this VisualElement? element, out T? parent) where T : VisualElement
 			=> TryFindParentElementWithParentOfType(element, out _, out parent);
+
+		public static async Task DisplayToastAsync(this VisualElement? visualElement, string message, int durationMilliseconds = 3000)
```

Comment:
Not sure if those methods should leave here because they are specific for SnackBar and not VisualElement

Reply:
But as you can see the extension method is for the VisualElement that is a anchor for the Snackbar, so I think it is in a right location

Reply:
TBH I feel that those methods should be in a `SnackbarExtensions` file because this is more related to Snackbar than a `VisualElement` thing.